### PR TITLE
enh(Header):  header and menu skeleton creates an unstable UI

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
**here is a video (speed:0.35):** 

https://user-images.githubusercontent.com/97687698/171838814-8ed531db-0da5-4fb6-8614-4d659b03ecd3.mp4

### CODE REVIEW

File: www/front_src/src/Header/PollerMenu/index.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { isEmpty, isNil } from 'ramda';
 import clsx from 'clsx';
 import { useAtomValue } from 'jotai/utils';
 import { useNavigate } from 'react-router-dom';
+import MenuLoader from 'centreon-frontend/packages/centreon-ui/src/MenuSkeleton';
```
Comment (Thebarda) [inline www/front_src/src/Header/PollerMenu/index.tsx:8]:
```suggestion
import { MenuSkeleton } from '@centreon/ui';
```

File: www/front_src/src/Header/RessourceStatusCounter/index.tsx [hunk 2]
```diff
@@ -3,14 +3,13 @@ import { useState, useRef, useEffect } from 'react';
 import axios from 'axios';
 import * as yup from 'yup';
 import { useAtomValue } from 'jotai/utils';
+import MenuLoader from 'centreon-frontend/packages/centreon-ui/src/MenuSkeleton';
```
Comment (Thebarda) [inline www/front_src/src/Header/RessourceStatusCounter/index.tsx:6]:
```suggestion
import { MenuSkeleton } from '@centreon/ui';
```

File: www/front_src/src/Header/userMenu/index.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { useTranslation, withTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 import { useUpdateAtom } from 'jotai/utils';
 import { gt, isNil, not, __ } from 'ramda';
+import MenuLoader from 'centreon-frontend/packages/centreon-ui/src/MenuSkeleton';
```
Comment (Thebarda) [inline www/front_src/src/Header/userMenu/index.tsx:8]:
```suggestion
import { MenuSkeleton } from '@centreon/ui';
```

File: www/front_src/src/Header/userMenu/index.tsx [hunk 4]
```diff
@@ -276,124 +279,128 @@ const UserMenu = (): JSX.Element => {
...
-                : `${t(
-                    labelPasswordWillExpireIn,
-                  )}: ${formattedPasswordRemainingTime}`
-            }
-          >
-            <Badge
-              color="warning"
-              invisible={passwordIsNotYetAboutToExpire}
-              variant="dot"
+    <div>
```
Comment (Thebarda) [inline www/front_src/src/Header/userMenu/index.tsx:282]: Do you need this `div` tag?

**Review by bdauria — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**